src/test/regress/pg_regress_multi.pl

```diff
@@ -56,6 +60,10 @@ ()
 my %functions = ();
 my %operators = ();
 my $valgrind = 0;
+my $valgrind_path = "valgrind";
+my $valgrind_log_file = "valgrind_test.log";
+my $pg_ctl_timeout = undef;
```

Comment:
[Here](https://github.com/citusdata/citus/pull/1294/files/97838113560ee66d8c0519d35e087d875b287e62#diff-a7ce4fb174e1acce25b5bcd3121576a9R153), we only set PGCTLTIMEOUT variable if pg_ctl_timeout is not defined. I think it is necessary to keep this as-is.